### PR TITLE
chore: add git-remote-codecommit manifest

### PR DESCRIPTION
**Pull request by ga-paul-t**

### CODE REVIEW

**Comment by bacongobbler** [comment]
Can you check the failing CI workflow?

**Comment by ga-paul-t** [comment]
Apologies, there was a typo in the manifest. Fixed it now

File: Food/git-remote-codecommit.lua [hunk 1]
```diff
@@ -0,0 +1,50 @@
+local name = "git-remote-codecommit"
+local version = "v1.0.0"
```
Comment (bacongobbler) [inline Food/git-remote-codecommit.lua:2]: `version` fields should not include a `v` in the string. Gofish uses this as part of its semantic version check.

Have a look at similar files like [bicep](https://github.com/fishworks/fish-food/blob/main/Food/bicep.lua) on how to accomplish this.

**Review by bacongobbler — CHANGES_REQUESTED** [review]

File: Food/git-remote-codecommit.lua (hunk 1 above)
Comment (ga-paul-t) [reply Food/git-remote-codecommit.lua]: I have followed the approach used by bicep. Thanks for letting me know

**Pull request merged**